papis/utils.py

```diff
@@ -71,9 +71,22 @@ def general_open(file_name: str,
         subprocess.call(cmd)
     else:
         logger.debug("Not waiting for process to finish")
-        subprocess.Popen(
-            cmd, shell=False,
-            stdin=None, stdout=None, stderr=None, close_fds=True)
+        popen_kwargs = {
+            "shell": False,
+            "stdin": None,
+            "stdout": None,
+            "stderr": None,
+            "close_fds": True
+        }
```

Comment:
```suggestion
        }	# type: Dict[str, Any]
```
You'll need to add that to make `mypy` happy.